src/networking/TranscriptJsonService.ts

```diff
@@ -2,14 +2,19 @@ import { FirebaseConfig } from "../app/AppConfigContext";
 import TranscriptJson from "../models/TranscriptJson";
 
 import ModelService from "./ModelService";
+import { COLLECTION_NAME } from "./PopulationOptions";
 
 export default class TranscriptJsonService extends ModelService {
   constructor(firebaseConfig: FirebaseConfig) {
-    super("TranscriptJson", firebaseConfig);
+    super(COLLECTION_NAME.Transcript, firebaseConfig);
   }
```

Comment:
Let's add TranscriptJson to `COLLECTION_NAME` and use that? Even though it's not a collection